app/views/admin/settings/index.html.slim

```diff
@@ -0,0 +1,27 @@
+- content_for(:title, 'Settings')
```

Comment:
@hooverlunch, 
Not sure if settings would be the right place for division-related functionalities. It would render the Manage Divisions page obsolete and I personally like the way it is right now. But, I can see that, at some point, we might need to add new stuff to Settings or even rethink the Manage Divisions page altogether. For the moment though, this is not going to be a priority.